recipes/threadpoolctl/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: threadpoolctl
+  version: {{ version }}
+
+source:
+  url: https://github.com/joblib/threadpoolctl/archive/{{ version }}.tar.gz
+  sha256: 49b987e9e86168eec68d0c706091bd6a8590268abc249b32bcd2a3909d331a04
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m flit install
+
+requirements:
+  host:
+    - python >=3.5
+    - flit
+  run:
+    - python >=3.5
+
+test:
+  requires:
+    - numpy
+  imports:
+    - threadpoolctl
+  commands:
+    - python -c "import numpy; from threadpoolctl import threadpool_info; threadpool_info()"
```

Comment:
Can you print the info as well?

Reply:
Should this just be moved to `run_test.py`? Would make it a bit easier to follow 🙂

Reply:
I added a `run_test.py`